Tests/Approval.Tests/ApprovedApi/FluentAssertions/net47.verified.txt

```diff
@@ -823,6 +824,7 @@ namespace FluentAssertions.Equivalency
         FluentAssertions.Equivalency.ConversionSelector ConversionSelector { get; }
         FluentAssertions.Equivalency.CyclicReferenceHandling CyclicReferenceHandling { get; }
         FluentAssertions.Equivalency.EnumEquivalencyHandling EnumEquivalencyHandling { get; }
+        bool ExcludeNonBrowsable { get; }
```

Comment:
I'm a little more worried about this one.
So I wanted to search the web a bit, to see if I can find someone who implements this interface

E.g. it _almost_ conflicts with https://github.com/fluentassertions/fluentassertions.json/pull/59/files#diff-cbc228d7e59fc36a3c426829ebf2bae9f6039116b210107e5506e15b96707aef

Reply:
The only implementors I can find on GitHub are people who have imported the FluentAssertions code whole-hog into their own codebase for whatever reason. They, of course, won't be seeing this change until/unless they upgrade their imported copy, which'll bring in the new implementations at the same time.

Reply:
I cloned the repos listed below and `grep`'ed for `IEquivalencyAssertionOptions`.
As none of those implemented the interface I'm okay with this.

```
atc-test/
atc/
FacilityCSharp/
FluentAssertions.ArgumentMatchers.Moq/
fluentassertions.aspnetcore.mvc/
FluentAssertions.AssertMultiple/
fluentassertions.autofac/
fluentassertions.bUnit/
FluentAssertions.Extensions/
FluentAssertions.Http/
fluentassertions.ioc.ninject/
fluentassertions.json/
FluentAssertions.LanguageExt/
fluentassertions.mvc/
FluentAssertions.NodaTime/
fluentassertions.reactive/
FluentAssertions.Web/
FluentResults/
Flushot/
functional-maybe-extensions/
Functional.Utilities/
Nut.Results/
playwright-dotnet-contrib/
RecShark/
RestVerifier/
SerilogSinksInMemory/
WireMock.Net/
xassert/
```

Reply:
Gee. I think you need to find a hobby 😅